src/main/resources/SnakemakeBundle.properties

```diff
@@ -38,6 +38,10 @@ INSP.NAME.section.multiline.string.args.message=Multiline string argument in ''{
 INSP.NAME.section.multiple.args=Only one argument is allowed for this section.
 INSP.NAME.section.multiple.args.message=Only one argument is allowed for ''{0}'' section.
 
+# SmkSectionMistypedArgsInspection
+INSP.NAME.section.unspecified.field.args=Field is not specified
+INSP.NAME.section.unspecified.field.args.message=No field from ''{0}'' is specified
```

Comment:
Could you change the message to smth more clear, so user could easily understand what `field` you are talking about and how to fix the bug. E.g. smth like: Expected rule section name or argument (e.g. 'rules.foo.input' or 'rules.foo.output.arg1'), but  rule/checkpoint/subworkflow object was found

Reply:
Done.